File: ZLERP.Web/Scripts/Views/CarAss/CarAss.js
function carassIndexInit(storeUrl) {
    var myJqGrid = new MyGrid({
        renderTo: 'MyJqGrid'
        //, width: '100%'
            , autoWidth: true
            , buttons: buttons0
            , height: gGridHeight
		    , storeURL: storeUrl
		    , sortByField: 'ID'
		    , primaryKey: 'ID'
		    , setGridPageSize: 30
		    , showPageBar: true
		    , initArray: [
                { label: '辅助单号', name: 'ID', index: 'ID', width: 90 }
                , { label: '任务单号', name: 'TaskID', index: 'TaskID', width: 90 }
                , { label: '工程名称', name: 'ProjectName', index: 'ProduceTask.ProjectName'  }
                , { label: '车辆代号', name: 'CarID', index: 'CarID', width: 90 }
                , { label: '车牌号', name: 'CarNo', index: 'Car.CarNo', width: 90 }
                , { label: '司机编号', name: 'UserID', index: 'UserID', width: 90 }
                , { label: '司机姓名', name: 'TrueName', index: 'User.TrueName', width: 90 }
                , { label: '出车时间', name: 'AssDate', index: 'AssDate', formatter: 'date', searchoptions: { dataInit: function (elem) { $(elem).datepicker(); }, sopt: ['ge']} }
                , { label: '辅助类型', name: 'AssType', index: 'AssType', formatter: dicCodeFmt, unformat: dicCodeUnFmt, formatoptions: { parentId: 'AssType'} }
                , { label: '趟次', name: 'AssTimes', index: 'AssTimes', width: 90 }
                , { label: '备注', name: 'Remark', index: 'Remark' }
                 , { label: '是否改变车辆状态', name: 'IsChangeCar', index: 'IsChangeCar', hidden: true }
		    ]
		    , autoLoad: true
            , functions: {
                handleReload: function (btn) {
                    myJqGrid.reloadGrid();
                },
                handleRefresh: function (btn) {
                    myJqGrid.refreshGrid('1=1');
                },
                handleAdd: function (btn) {
                    myJqGrid.handleAdd({
                        loadFrom: 'MyFormDiv',
                        btn: btn
                    });
                },
                handleEdit: function (btn) {
                    myJqGrid.handleEdit({
                        loadFrom: 'MyFormDiv',
                        btn: btn
                    });
                }
                , handleDelete: function (btn) {
                    myJqGrid.deleteRecord({
                        deleteUrl: btn.data.Url
                    });
                }
            }
    });

}

File: ZLERP.Model/CarAss.cs
using System;
using System.Collections.Generic;
using System.Text; 
using ZLERP.Model.Generated;
using System.ComponentModel;
using System.ComponentModel.DataAnnotations;
namespace ZLERP.Model
{
    /// <summary>
    /// 司机辅助作业
    /// </summary>
	public class CarAss : _CarAss
    {
        /// <summary>
        /// 车辆代号
        /// </summary>
        [DisplayName("车辆代号")]
        [Required]
        public virtual string CarID
        {
            get;
            set;
        }

        /// <summary>
        /// 司机编号
        /// </summary>
        [DisplayName("司机编号")]
        [Required]
        public virtual string UserID
        {
            get;
            set;
        }

        public virtual string TrueName
        {
            get { return this.User == null ? "" : this.User.TrueName; }
        }

        public virtual string CarNo
        {
            get { return this.Car == null ? "" : this.Car.CarNo; }
        }

        /// <summary>
        /// 任务单号
        /// </summary>
        [DisplayName("任务单号")]        
        public virtual string TaskID
        {
            get;
            set;
        }

        public virtual string ProjectName
        {
            get { return this.ProduceTask == null ? "" : this.ProduceTask.ProjectName; }
        }

        [DisplayName("是否改变车辆状态")]
        public virtual bool IsChangeCar
        {
            get;
            set;
        }

        /// <summary>
        /// 辅助单号
        /// </summary>
        [DisplayName("辅助单号")]
        public override string ID
        {
            get
            {
                return base.ID;
            }
            set
            {
                base.ID = value;
            }
        }
	}
}